core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -83,6 +86,21 @@ public abstract class LoggingDecoratorBuilder {
 
     private boolean buildLogWriter;
 
+    /**
+     * Creates a new instance.
+     */
+    protected LoggingDecoratorBuilder() {
+        defaultLogger = null;
+    }
```

Comment:
Could we remove this constructor?

Reply:
Added it for backward compatibility, but let me remove this and use a method for setting the default logger.